Documentation/DomainModel/Entities.md

```diff
@@ -0,0 +1,209 @@
+# Domain Entities
+
+## Identity Context
+
+### Administrator
+Table name: `phplist_admin`
+
+An **administrator** can log in to the system and is allowed to administer
+selected **lists** (as the owner), send **campaigns** to these lists and edit
+**subscribers**.
+
+Administrators are not subscribers. If administrators would like to subscribe
+to lists, they need to have a separate subscriber account.
+
+### AdministratorAttribute
+Table name: `phplist_adminattribute` or `phplist_admin_attribute` 
+
+What does this do? How does this work?
+
+### AdministratorPasswordRequest
+Table name: `phplist_admin_password_request`
+
+What does this do? How does this work?
+
+### AdministratorToken
+Table name: `phplist_admintoken`
+
+This table contains the API tokens for **administrators**. Those API tokens are
+used for access to the REST API.
+
+
+## SubscriptionContext
+
+### Attribute
+Table name: `phplist_user_attribute`
+
+An **attribute** is a field for subscribers. This entity does not
+contain the values for this attribute for each individual subscribe, but
+only the name of the attribute and an ID. 
+
+### AttributeValue
+Table name: `phplist_user_user_attribute`
+
+An **attributeValue** contains the value for a particular **attribute** for a
+particular **subscriber**.
+
+### SubscribePage
+Table name: `phplist_subscribepage`
+
+What does this do? How does this work?
```

Comment:
subscribe pages allow setting up a selection of lists, attributes and language, and some other settings to control the content for the page that can be used to subscribe to the system. As a result, you can eg have different pages per language, which allows you to translate all the content for each language.